android/com/jsyn/devices/android/AndroidAudioForJSyn.java

```diff
@@ -84,9 +84,9 @@ public void start() {
             minBufferSize = AudioTrack.getMinBufferSize(frameRate,
                     AudioFormat.CHANNEL_OUT_STEREO,
                     AudioFormat.ENCODING_PCM_FLOAT);
-            System.out.println("Audio minBufferSize = " + minBufferSize);
+            LOGGER.debug("Audio minBufferSize = " + minBufferSize);
```

Comment:
Please stick to the org.slf4j.Logger since this is the most compatible way of using logging on multiple platforms including android with all its features.

Reply:
This was never fixed. The line should have been removed.